ros/processor/inventory_events_consumer.py

```diff
@@ -101,8 +101,8 @@ def host_delete_event(self, msg):
         host_id = msg['id']
         insights_id = msg['insights_id']
         with app.app_context():
-            LOG.info(
-                '%s - Deleting performance profile records with insights_id %s',
+            LOG.debug(
```

Comment:
Don't know why this log was added, but do we really need it.  Also, same for `if rows_deleted > 0:` [here](https://github.com/RedHatInsights/ros-backend/blob/main/ros/processor/inventory_events_consumer.py#L111). Should we replace `>0` with `==1` as only one unique host would be deleted for every delete event ,I suppose. Any thoughts?

Reply:
Made condition using `==1`.

Reply:
Info level is not required for this log statement while debugging I think it will be helpful.